paper-api/src/main/java/org/bukkit/block/DecoratedPot.java

```diff
@@ -74,4 +81,12 @@ public static enum Side {
         RIGHT,
         FRONT
     }
+
+    /**
+     * An animation style of decorated pot.
```

Comment:
sounds a bit rough
```suggestion
     * Style a DecoratedPot can wobble in.
```

^ maybe someone has a better idea for jdocs but this sounds a bit better.